addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -552,7 +552,7 @@
 		
 		"1103"	//Back Scatter
 		{
-			"desp"			"Back Scatter: {positive}Fires fast rockets instead of bullets, crits when aiming at boss, {negative}deals less damage"
+			"desp"			"Back Scatter: {positive}Crits when aiming directly at the boss, {neutral}fires fast rockets instead of bullets"
```

Comment:
Does make more sense to have neutral first before positive for this and anywhere else